app/views/membership.html

```diff
@@ -193,7 +193,9 @@
                             <span ng-bind="newBinding.namespace"></span>
                           </ui-select-match>
                           <ui-select-choices
-                            repeat="projectName in projects | filter: $select.search">
+                            repeat="projectName in projects track by $index | filter: $select.search"
```

Comment:
`track by $index` doesn't look right here and below, particularly since we're calling `unshift` to prepend things.